setup.py

```diff
@@ -112,9 +85,7 @@ def run(self):
     },
 
     install_requires=[
-        # Any changes here must be reflected in ``python-flocker.spec.in`` so
-        # that RPM dependencies match.
-        # They must also be reflected in the yumdownloader lines in
+        # Any changes here must be reflected in the yumdownloader lines in
```

Comment:
We do have some zfs packages in there unfortunately Which I guess should probably be listed there, but aren't currently. :frowning: That is my fault, I guess. Although, those aren't the steps that were used to upload those zfs packages. I guess that means that section can probably just go.